src/StockComparison.js

```diff
@@ -0,0 +1,97 @@
+const comparisonStyles = {
+  display: "flex",
+  flexDirection: "column",
+};
+
+const bodyStyles = {
+  display: "flex",
+  flexDirection: "row",
+  wrap: "nowrap",
+  justifyContent: "space-around",
+};
+
+const gifStyle = {
+  height: "150px",
+  // width: "100px",
+};
+
+const StockComparison = ({
+  name,
+  symbol,
+
+  historicalPrice = "$123.24",
+  historicalLow = "$104.25",
+  historicalHigh = "$129.25",
+
+  couldHaveMadeBank = false,
```

Comment:
How would these values be calculated? is historicalPrice, Low, High being passed already "pretified" or should that be done when displaying the info?